tests/InputFiles/Cce/cce_lmax_12_CceR0257.yaml

```diff
@@ -0,0 +1,36 @@
+# Distributed under the MIT License.
```

Comment:
Please make the file name CamelCase. You probably don't need to include as much data in the file name since it's an example and used to test-run the executable on CI, `CharacteristicExtract.yaml` should suffice.

Reply:
Also rename the observation file name please